kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -302,27 +230,21 @@ log_level -i "PUBLICIP_DNS:                             $PUBLICIP_DNS"
 log_level -i "PUBLICIP_FQDN:                            $PUBLICIP_FQDN"
 log_level -i "REGION_NAME:                              $REGION_NAME"
 log_level -i "RESOURCE_GROUP_NAME:                      $RESOURCE_GROUP_NAME"
+log_level -i "SPN_CLIENT_ID:                            ----"
+log_level -i "SPN_CLIENT_SECRET:                        ----"
 log_level -i "SSH_PUBLICKEY:                            ----"
 log_level -i "STORAGE_PROFILE:                          $STORAGE_PROFILE"
 log_level -i "TENANT_ID:                                $TENANT_ID"
 log_level -i "TENANT_SUBSCRIPTION_ID:                   $TENANT_SUBSCRIPTION_ID"
```

Comment:
Print values which are now getting passed. For ex: 
"aksEngineBaseURL"
      "aksEngineReleaseVersion"
      "galleryRepository"
      "galleryBranch"
      "clusterDefinitionFileName"